### PR TITLE
Update README.md for Helm Chart Link

### PR DESCRIPTION
**Pull request by inichols**
**What this PR does / why we need it**:

Installation & Usage is pointing to a broken page for deployed as a Helm Chart. Believe this is the page that is meant to be linked

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Currently, this project has the following functionality:
 
 ## Installation & Usage
 
-Rancher Turtles is deployed using a Helm Chart. For instructions, see the [getting started guide](https://turtles.docs.rancher.com/docs/category/getting-started).
+Rancher Turtles is deployed using a Helm Chart. For instructions, see the [getting started guide](https://turtles.docs.rancher.com/getting-started/install-rancher-turtles/using_helm).
```
Comment (salasberryfin) [inline README.md:20]:
```suggestion
Rancher Turtles is deployed using a Helm Chart. For instructions, see the [installation guide](https://turtles.docs.rancher.com/category/install-rancher-turtles).
```

**Review by salasberryfin — CHANGES_REQUESTED** [review]
Hi @inichols, thanks for reporting this issue. Since we consider the installation via Rancher Dashboard as preferred alternative, I'd say we should point this link to the [Install Rancher Turtles](https://turtles.docs.rancher.com/category/install-rancher-turtles) homepage, which includes links for both alternatives: via UI and via terminal. What do you think?

**Comment by furkatgofurov7** [comment]
@inichols hey, thanks for the PR! Can you resolve the comment from @salasberryfin please, so we can take this PR in?

**Comment by furkatgofurov7** [comment]
Duplicate of #602

**Pull request closed without merging**